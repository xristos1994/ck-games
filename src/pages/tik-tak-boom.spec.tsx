import React from 'react';
import renderer from 'react-test-renderer';
import TikTakBoomPage from './tik-tak-boom';
import EL_TikTakBoomPage from './el/tik-tak-boom';
import EN_TikTakBoomPage from './en/tik-tak-boom';

const TikTakBoom = () => {
  return (
    <div>
      TikTakBoom
    </div>
  );
};

jest.mock('@page-components/', () => ({
  __esModule: true,
  TikTakBoom: TikTakBoom
}));

describe('pages/Home', () => {
  it('renders correctly', () => {
    let tree = renderer.create(
      <TikTakBoomPage />
    ).toJSON();
    expect(tree).toMatchSnapshot();

    tree = renderer.create(
      <EL_TikTakBoomPage />
    ).toJSON();
    expect(tree).toMatchSnapshot();

    tree = renderer.create(
      <EN_TikTakBoomPage />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
