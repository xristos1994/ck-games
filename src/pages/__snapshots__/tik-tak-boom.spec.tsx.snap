// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/Home renders correctly 1`] = `
<div>
  TikTakBoom
</div>
`;

exports[`pages/Home renders correctly 2`] = `
<div>
  TikTakBoom
</div>
`;

exports[`pages/Home renders correctly 3`] = `
<div>
  TikTakBoom
</div>
`;
